src/docbkx/chapter13-getting-started-monitor.xml

```diff
@@ -249,7 +249,7 @@
 					scalefit="1" />
 			</imageobject>
 		</mediaobject>
-		<para>Note that the contents of the dialog is highly dependent on the
+		<para>Note that the content of the dialog is highly dependent on the
```

Comment:
I believe contents is correct. Like contents of a box.

http://english.stackexchange.com/questions/13556/content-or-contents